source/common.ts

```diff
@@ -357,7 +357,7 @@ const i18n = {
     const { close, defaultLocale, hideMissing, open } = i18n.options;
     const {
       _locale: locale = i18n.getLocale(),
-      purify = i18n.options.purify,
+      _purify = i18n.options.purify,
```

Comment:
I'd rename it in here:
```suggestion
      _purify: purify = i18n.options.purify,
```